File: CPP/XOR/XOR Queries of a Subarray.cpp
class Solution {
public:
    vector<int> xorQueries(vector<int>& arr, vector<vector<int>>& queries) {
        vector<int>ans;
        for(auto i:queries){
            int op = 0;
            for(int j = i[0];j<=i[1];j++){
                op=op^arr[j];
            }
            ans.push_back(op);
        }
        return ans;
    }
};

File: Python/XOR/XOR Queries of a Subarray.py
class Solution:
    def xorQueries(self, arr: List[int], queries: List[List[int]]) -> List[int]:
        ans = list()
        for i in queries:
            xorSum  = 0
            # j = i[0]
            # while j<=i[1]:
            #     xorSum=xorSum^arr[j]
            #     j+=1
            # ans.append(xorSum)
            for j in range(i[0],i[1]+1):
                xorSum=xorSum^arr[j]
            ans.append(xorSum)
        return ans
                
        
